test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -1496,24 +1508,36 @@ public void queryClusterStatus() throws Exception {
 
         // STEP 1.
         ClusterStatusReport clusterStatus = getCorfuRuntime().getManagementView().getClusterStatus();
-        Map<String, NodeStatus> nodeStatusMap = clusterStatus.getClientServerConnectivityStatusMap();
+        Map<String, ConnectivityStatus> nodeConnectivityMap = clusterStatus.getClientServerConnectivityStatusMap();
+        Map<String, NodeStatus> nodeStatusMap = clusterStatus.getClusterNodeStatusMap();
+        ClusterStatusReport.ClusterStatusReliability clusterStatusReliability = clusterStatus.getClusterStatusReliability();
+        assertThat(nodeConnectivityMap.get(SERVERS.ENDPOINT_0)).isEqualTo(ConnectivityStatus.RESPONSIVE);
+        assertThat(nodeConnectivityMap.get(SERVERS.ENDPOINT_1)).isEqualTo(ConnectivityStatus.RESPONSIVE);
+        assertThat(nodeConnectivityMap.get(SERVERS.ENDPOINT_2)).isEqualTo(ConnectivityStatus.RESPONSIVE);
         assertThat(nodeStatusMap.get(SERVERS.ENDPOINT_0)).isEqualTo(NodeStatus.UP);
         assertThat(nodeStatusMap.get(SERVERS.ENDPOINT_1)).isEqualTo(NodeStatus.UP);
         assertThat(nodeStatusMap.get(SERVERS.ENDPOINT_2)).isEqualTo(NodeStatus.UP);
+        assertThat(clusterStatusReliability).isEqualTo(ClusterStatusReport.ClusterStatusReliability.STRONG_QUORUM);
         assertThat(clusterStatus.getClusterStatus()).isEqualTo(ClusterStatus.STABLE);
 
         // STEP 2.
```

Comment:
As it's seen in this test, with the new implementation is possible to have Server Nodes with status UP but connectivity status UNRESPONSIVE. Maybe it will be good to have a comment in this code explaining this.

Reply:
Done.